api/prisma/schema.prisma

```diff
@@ -127,13 +121,20 @@ model user {
   portfolio                    Portfolio[]
   profileUI                    ProfileUI? // Undefined
   progressTimestamps           Json? // ProgressTimestamp[] | Null[] | Int64[] | Double[] - TODO: NORMALIZE
-  // rand Float? // Undefined removed, as new API never uses it.
+  /// A random number between 0 and 1.
+  ///
+  /// Valuable for selectively performing random logic.
+  rand                         Float?
   savedChallenges              SavedChallenge[] // Undefined | SavedChallenge[]
   sendQuincyEmail              Boolean
   theme                        String? // Undefined
   timezone                     String? // Undefined
   twitter                      String? // Null | Undefined
   unsubscribeId                String
+  /// Used to track the number of times the user's record was written to.
+  ///
+  /// This has the main benefit of allowing concurrent ops to check for race conditions.
+  updateCount                  Int                           @default(0)
```

Comment:
```suggestion
  updateCount                  Int?                          @default(0)
```
Old records won't have this field.

Reply:
I thought that was the point. When this field does not exist in MongoDB, accessing it defaults to `0`?

Reply:
That's the bug I mentioned. https://github.com/prisma/prisma/issues/24680

At the very least it's inconsistent.

Anyways, adding the ? means that the type is `null | number`, which is correct.